server/svix-server/src/queue/redis.rs

```diff
@@ -36,17 +42,29 @@ use super::{
 
 // FIXME: Change unwraps to have our own error type for the queue module entirely
 
+const MAIN: &str = "svix_{queue}_v3_main";
+
+// TODO: Stream based delayed queue
+// const DELAYED: &str = "svix_queue_v3_delayed";
+
 // FIXME: hash-tags here enable clustering but prevent proper sharding of lists
-const MAIN: &str = "{queue}_svix_main";
-const PROCESSING: &str = "{queue}_svix_processing";
-const DELAYED: &str = "{queue}_svix_delayed";
+const LEGACY_MAIN: &str = "{queue}_svix_main";
+const LEGACY_PROCESSING: &str = "{queue}_svix_processing";
+const LEGACY_DELAYED: &str = "{queue}_svix_delayed";
 
-const LEGACY_MAIN: &str = "svix_queue_main";
-const LEGACY_PROCESSING: &str = "svix_queue_processing";
-const LEGACY_DELAYED: &str = "svix_queue_delayed";
+const LEGACY_LEGACY_MAIN: &str = "svix_queue_main";
+const LEGACY_LEGACY_PROCESSING: &str = "svix_queue_processing";
+const LEGACY_LEGACY_DELAYED: &str = "svix_queue_delayed";
 
-/// After this limit a task should be taken out of the processing queue and rescheduled
-const TASK_VALIDITY_DURATION: Duration = Duration::from_secs(45);
+/// Consumer group name constant
+const WORKERS_GROUP: &str = "svix_workers";
+/// Consumer group consumer name constant
+const WORKER_CONSUMER: &str = "svix_worker";
```

Comment:
The string values are too similar for vastly different things. You need to update the string value.
It's weird that one of the vars is plural and the other is singular.